src/app/shared-component/dialog/dialog.component.ts

```diff
@@ -39,4 +40,22 @@ export class DialogComponent implements OnInit {
     this.onClick.emit(val);
     this.modalFadeIn = false;
   }
+
+  @HostListener('window:keydown', ['$event'])
+  onActionKeyPress(event: any): void {
+    let currentDefaultIndex = this.dialog.actionButtons.findIndex(i => i.default);
```

Comment:
The reason for the error is that the class ActionItem is a private class and it's Type can't be seen by TypeScript. This makes TS think that "default" is of Type Object. To fix this, you just need to "export" the class.